packages/create-subscription/package.json

```diff
@@ -15,7 +15,7 @@
     "cjs/"
   ],
   "peerDependencies": {
-    "react": "^17.0.0-alpha"
+    "react": "17.0.0-rc.0"
```

Comment:
I made them all strict. I think going forward we might want to keep it (and not just for RCs).